test/routes/test_post_route.py

```
@@ -51,43 +45,22 @@ def test_create_post_success(client: TestClient, db_session: Session, setup_user
+    
+    assert "hashtag" in data
+    assert len(data["hashtag"]) == 2
+    assert data["hashtag"][0]["word"] == "python"
+    assert data["hashtag"][1]["word"] == "fastapi"
 
-def test_create_post_no_images(client: TestClient, db_session: Session, setup_user):
-    """
-    성공 케이스: 
-    - 이미지가 없는 경우에도 생성되어야 함
-    """
+def test_create_post_too_few_images(client: TestClient, db_session: Session, setup_user):
```

Ruff (0.14.5)

Unused function argument: `db_session`

(ARG001)